### PR TITLE
[Pod] Add Initial Podspec

### PR DESCRIPTION
**Pull request by Ben-G**
## Changes in this pull request

Adds an initial podspec that allows development of the `ReactiveLists-ReactiveSwift` extension.

### CODE REVIEW

File: ReactiveLists.podspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  s.description      = <<-DESC
+                        React-like API for UITableView and UICollectionView 
+                        DESC
+  s.homepage         = "https://github.com/plangrid/ReactiveLists"
+  s.license          = { :type => "MIT", :file => "LICENSE.md" }
+  s.author           = "PlanGrid"
+  s.documentation_url = "https://plangrid.github.io/ReactiveLists"
+  s.social_media_url = "https://medium.com/plangrid-technology"
+  s.source           = { :git => "https://github.com/plangrid/ReactiveLists", :tag => s.version.to_s }
+  s.ios.deployment_target     = '8.0'
```
Comment (jessesquires) [inline ReactiveLists.podspec]: we're 9+

(actually using some 9+ APIs)

File: ReactiveLists.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  s.version          = "0.0.1"
+  s.summary          = "React-like API for UITableView and UICollectionView"
+  s.description      = <<-DESC
+                        React-like API for UITableView and UICollectionView 
+                        DESC
+  s.homepage         = "https://github.com/plangrid/ReactiveLists"
+  s.license          = { :type => "MIT", :file => "LICENSE.md" }
+  s.author           = "PlanGrid"
+  s.documentation_url = "https://plangrid.github.io/ReactiveLists"
+  s.social_media_url = "https://medium.com/plangrid-technology"
```
Comment (jessesquires) [inline ReactiveLists.podspec]: should this be twitter?

File: ReactiveLists.podspec [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name             = "ReactiveLists"
+  s.version          = "0.0.1"
+  s.summary          = "React-like API for UITableView and UICollectionView"
+  s.description      = <<-DESC
+                        React-like API for UITableView and UICollectionView 
+                        DESC
```
Comment (jessesquires) [inline ReactiveLists.podspec]: never really got this syntax.

can't this also just be `s.description = "React-like API for UITableView and UICollectionView"` ?

File: ReactiveLists.podspec [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  s.license          = { :type => "MIT", :file => "LICENSE.md" }
+  s.author           = "PlanGrid"
+  s.documentation_url = "https://plangrid.github.io/ReactiveLists"
+  s.social_media_url = "https://medium.com/plangrid-technology"
+  s.source           = { :git => "https://github.com/plangrid/ReactiveLists", :tag => s.version.to_s }
+  s.ios.deployment_target     = '8.0'
+  s.requires_arc = true
+  s.source_files     = 'Sources/**/*.swift'
+  s.dependency 'Dwifft', '~> 0.7.0'
+end
```
Comment (jessesquires) [inline ReactiveLists.podspec]: newline plz 😁

File: ReactiveLists.podspec (hunk 2 above)
Comment (Ben-G) [reply ReactiveLists.podspec]: Hm..not sure.. as there's nothing tech related being shared via twitter (all business)

File: ReactiveLists.podspec (hunk 3 above)
Comment (Ben-G) [reply ReactiveLists.podspec]: Syntax for multi-line. But with brief description agree that plain string is better.

**Comment by Ben-G** [comment]
@jessesquires Feedbacksss addressed!

**Pull request merged**